pep-9999.rst

```diff
@@ -47,13 +47,57 @@ required to interpret all standard constructs, and additionally a type checker
 can support extra constructs not described in this PEP. All deviations from
 this PEP should be documented.
 
+Syntax and Directory Structure
+==============================
+
+Type stubs are syntactically valid Python files with a ``.pyi`` suffix.
+Type checkers are expected to support the syntax of the latest Python
+version. Type stubs should be parseable with the latest released Python
+version, but also support the previous version up until six months after
```

Comment:
I think the usage of "support" here is confusing, since it refers only to syntactic validity. Probably better to just consistently say "should be syntactically valid" here.

Reply:
Also, it might be good to have an explicit example of a timeline that would have applied to old features (like "December 1, 2016 - 3.6 is released; May 1, 2017 - stubs may now use variable annotations").

Reply:
(I didn't look up the actual dates though.)